web_monitoring/html_diff_render.py

```diff
@@ -772,7 +1006,9 @@ def merge_changes(change_chunks, doc, tag_type='ins'):
         # FIXME: explicitly handle elements that can't have our markers as
         # direct children.
         if chunk[0] == '<':
-            name = chunk.split()[0].strip('<>/')
+            # NOTE: split first by `>` because we could have undiffable items
+            # with content here, i.e. more than one tag.
+            name = chunk.split('>')[0].split()[0].strip('<>/')
```

Comment:
Since we only care about the first element of `chunk.split('>')`, we can use `chunk.split('>', 1)` to save time. Same for `split()[0]` -> `split(None, 1)`.